game-core/src/main/java/games/strategy/triplea/ui/screen/TileManager.java

```diff
@@ -376,7 +378,8 @@ private void drawUnits(final Territory territory, final MapData mapData, final S
     }
   }
 
-  private static List<UnitCategory> getSortedUnitCategories(final Territory t, final GameData data) {
+  @VisibleForTesting
+  public static List<UnitCategory> getSortedUnitCategories(final Territory t, final GameData data) {
```

Comment:
I think making this method package private should be sufficient

Reply:
Yeah, updated.